lib/yabeda/datadog.rb

```diff
@@ -12,6 +13,7 @@ module Datadog
 
     # TODO: consider to change too manual
     def self.start
+      raise ApiKeyError, 'DataDog API key or application key not set to envoirmental variable' if ENV['DATADOG_API_KEY'].nil? || ENV['DATADOG_APP_KEY'].nil?
```

Comment:
It will also be great if you follow the Rubocop rules aka "Cops". For example this line exceeds the maximum length of 120 characters.

You can learn about Rubocop [here](https://github.com/rubocop-hq/rubocop)